fixtures/fixtures.js

```diff
@@ -55,12 +55,14 @@ var Fixtures = {
         ]
     },
     voucherStatus: {
-        "success":
-            {"success":["RVNT12345678"], "fail":[], "invalid":[]},
-        "fail":
-            {"success":[], "fail":["FAL11111111"], "invalid":[]},
-        "invalid":
-            {"success":[], "fail":[], "invalid":["INV1"]},
+        "success": [{"message": 'Voucher is valid'}],
+        "warning": [
+            {"warning": "You have already submitted voucher code :code."},
```

Comment:
In a fixture? possibly not - It would be nice to echo the actual number asked for here, but i'm not sure the fixture knows what that would be. I'd card it for long grass?